server/log_operation_manager.go

```diff
@@ -197,6 +197,9 @@ type LogOperationManager struct {
 	tracker        *util.MasterTracker
 	heldMutex      sync.Mutex
 	lastHeld       []int64
+	// Cache of logID => name; assumed not to change during runtime
+	nameMutex sync.Mutex
```

Comment:
nit: Maybe `logNamesMutex`, so it mirrors the field name?

Reply:
Done